docs/release-notes.md

```diff
@@ -4,17 +4,28 @@ nav_order: 9
 
 # Release Notes
 
-## Upcoming Ignition 2.17.0 (unreleased)
+## Upcoming Ignition 2.17.1 (unreleased)
 
 ### Breaking changes
 
+### Features
+
+### Changes
+
+### Bug fixes
+
+## Ignition 2.17.0 (2023-11-24)
+
+Starting with this release, ignition-validate binaries are signed with the
+[Fedora 39 key](https://getfedora.org/security/).
 
 ### Features
 
 - Support the native Apple Hypervisor
 - Support Hetzner Cloud
-- A GRUB configuration suitable for use with https://github.com/coreos/bootupd
-  can now be installed; use `make install-grub-for-bootupd` to install it
+- A GRUB configuration suitable for use with https://github.com/coreos/bootupd can now be installed; 
```

Comment:
I believe we want 80 characters per line max.

and the second line seems to have lost formatting.